xyz/vector.go

```diff
@@ -0,0 +1,28 @@
+package xyz
+
+import (
+	"github.com/twpayne/go-geom"
+	"math"
+)
```

Comment:
nit: It's nice to order imports as
```go
import (
  // stdlib packages

  // third-party packages
)
```
i.e.
```go
import (
    "math"

    "github.com/twpayne/go-geom"
)
```
I suggest that we clean this up in a separate pull request.